src/Models/Database.php

```diff
@@ -33,4 +33,21 @@ public function insertUserDetails($name, $phone){
         $ans = $query->get_result();
         return $ans;
     }
+
+    // query for inserting block 
+    public function insertBlockDetails($title, $desc) {
+        $query = $this->conn->prepare("INSERT INTO customBlock(block_title, block_desc) VALUES('$title', '$desc')");
```

Comment:
We are not correctly using the prepared statement by inserting variables inside a string. can we properly use the prepared statement and pass data variables correctly